lib/iris/cube.py

```diff
@@ -2023,6 +2196,16 @@ def vector_summary(vector_coords, cube_header, max_line_offset,
                 summary += '\n     Cell Measures:\n'
                 summary += '\n'.join(cell_measure_summary)
 
+            #
+            # Generate summary of cube ancillary variables attribute
+            #
+            if vector_ancillary_variables:
+                ancillary_variable_summary, cube_header = vector_summary(
+                    [], cube_header, max_line_offset,
+                    ancillary_variables=vector_ancillary_variables)
+                summary += '\n     Ancillary Datasets:\n'
```

Comment:
I'd argue it should go the other way; we should update the other section headings (`Dimension coordinates -> Dimension Coordinates`) since they are headings and so should be capitalised

Reply:
FWIW I lean toward @lbdreyer here : "All Title Case" makes sense to me.

Reply:
Should this say `Ancillary Variables?`.